orttraining/orttraining/test/python/orttraining_test_ortmodule_onnx_ops.py

```diff
@@ -79,7 +79,7 @@ def run_step(model, x):
         for onnx_model in [onnx_graph_inf, onnx_graph_train]:
             for oimp in onnx_model.opset_import:
                 if oimp.domain == "":
-                    self.assertEqual(oimp.version, 15)
+                    self.assertEqual(oimp.version, 15)  # TODO: does this need updating??
```

Comment:
yes you need to update the version here to the default version which is now 20. Please add a comment to make this clear in future.